### ISSUE TITLE
Role of [] in array "literals"

### ISSUE THREAD

**Issue by 2colours** [Raku/doc#4172]
## Problem or new feature

It's a common gotcha that `[[foo]]` is the same as `[foo]`. Even though I don't like this, I can't foresee it changing so it would be better to clarify it.

## Suggestions

https://docs.raku.org/language/operators#circumfix_[_]  could be clearer that it's mainly a conversion subroutine, not some syntactically enforced enclosing of array items. Also, I think this could deserve an entry in the traps section.

**Comment by codesections** [comment]
I agree with putting this in traps.  We should probably mention that the syntax to dwim with `[[foo]]` is `[[foo],]`.

**Comment by 2colours** [comment]
yes - by the way, `[$[foo]]` can also work :)
